Tearing_test/TearingEngine_test.cpp

```diff
@@ -70,7 +70,7 @@ class TearingEngine_test : public BaseSimulationTest
     void TearDown() override
     {
         if (m_instance.root != nullptr)
-            sofa::simulation::getSimulation()->unload(m_instance.root);
+            sofa::simulation::node::unload(m_instance.root);
```

Comment:
I do not know what is this change about.

Reply:
This is the propagation from change done in the SOFA core code.